style.css

```diff
@@ -538,4 +570,59 @@
     height: 1px;
     margin-right: 5vw;
   }
+
+  .contact-form {
+    background-image: url("images/contact\ form\ shapes-mobile@2x.png");
+    background-repeat: no-repeat;
+    background-size: 60%;
+    background-position: right;
+    height: 779px;
+    background-color: #6070ff;
+    padding: 1rem 0 3rem 0;
+    border-top-left-radius: 100px;
+  }
+
+  .contact-form form {
+    display: flex;
+    flex-direction: column;
+    font-family: var(--font-family);
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 20px;
+    width: 327px;
+    height: 400px;
+  }
+
+  #name,
+  #email {
```

Comment:
kindly add a padding left to all placeholder text in your inputs 😊
**your design:**

![placeholder padding left  student](https://user-images.githubusercontent.com/59707859/220474767-b33f299f-b9c5-4f20-b811-35b0c046b2ef.PNG)

**figma design:**

![placeholder padding left](https://user-images.githubusercontent.com/59707859/220474771-c2c81f16-3e70-4000-95f2-141a67e338e1.PNG)